src/rt-app_parse_config.c

```diff
@@ -785,6 +785,30 @@ static void parse_cpuset_data(struct json_object *obj, cpuset_data_t *data)
 	log_info(PIN "key: cpus %s", data->cpuset_str);
 }
 
+static void parse_numa_data(struct json_object *obj, numaset_data_t *data)
+{
+	data->numaset = NULL;
+	data->numaset_str = get_string_value_from(obj, "nodes", TRUE, NULL);
+
+	if(data->numaset_str != NULL) {
+#if HAVE_LIBNUMA
```

Comment:
Where does HAVE_LIBNUMA get set?

Reply:
In configure.ac file I added line:
AC_CHECK_LIB([numa], [numa_available])
It takes care of library links and compile flags